substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationFiles.java

```diff
@@ -96,6 +96,9 @@
 
         @Option(help = "Comma-separated list of file names with declarative substitutions", type = OptionType.User)//
         public static final HostedOptionKey<String[]> SubstitutionFiles = new HostedOptionKey<>(null);
+
+        @Option(help = "Skip processing configuration files", type = OptionType.Expert)
+        public static final HostedOptionKey<Boolean> SkipConfigurationFiles = new HostedOptionKey<>(false);
```

Comment:
`HostedOptions` are only used for option processing in the image-builder and not the driver executable. If you want some driver specific option you have to add to `com.oracle.svm.driver.DefaultOptionHandler#consume`.

Also what is the usecase of this option? cc @christianwimmer

Reply:
> HostedOptions are only used for option processing in the image-builder and not the driver executable. If you want some driver specific option you have to add to com.oracle.svm.driver.DefaultOptionHandler#consume.

+1

> Also what is the usecase of this option? cc @christianwimmer

Spelled out in linked issue #2535

Reply:
But this patch is a all-or-nothing approach and does not allow to exclude native image properties processing on a finer granular scale as suggested in https://github.com/oracle/graal/issues/2535#issuecomment-638208260
Instead there could be an option that takes a regex and uses that to decide if a given native image properties resource location should be excluded or not.

Reply:
I am OK to add this as an non-API option, but then one needs to be able to specify the jars that are ignored. I don't believe there is much general value if one skips all jars on the classpath.

Reply:
I agree that skipping all configuration files is not really useful. It does not solve the "compositing" problem.

According to https://github.com/quarkusio/quarkus/issues/1658#issuecomment-762742950 the reason for this PR is the Oracle JDBC driver containing `--initialize-at-run-time=oracle.jdbc.OracleDriver,java.sql.DriverManager`. Since class initialization at run time is now the default, it should not be necessary at all.

Reply:
Thx @christianwimmer @vjovanov @olpaw for the feedback. I've started a [discussion thread]((https://graalvm.slack.com/archives/CN9KSFB40/p1611664256033900)) in the GraalVM slack channel, where we can hash out an improved solution.